scripts/build/build-config.sh

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
-echo "build-config: running medic-conf"
+echo "build-config: running cht-conf"
 
-node ./node_modules/medic-conf/src/bin/medic-conf.js \
+node ./node_modules/cht-conf/src/bin/index.js \
```

Comment:
```suggestion
node ./node_modules/.bin/cht \
```

Minor, but one of the things I picked up from the `check-app-settings-is-committed`  files that might make this a bit cleaner (and less dependent on the position/naming of source files....)